### PR TITLE
Allow to split emojis in the tokenstream and other fixes

### PR DESCRIPTION
**Pull request by zeehio**
This PR is divided in 3 independent commits:

- Move lang independent features out of usenglish
- Add copyright headers to two files
- Simplify tokenstream detection of emoji

### Move lang independent features out of usenglish

Some of the HTS features can be shared among voices from different languages but they were placed under the `us_ffeatures.c` file. This commit moves them to the common `cst_ffeatures.c` file so other languages can use them as well. This is required for multilingual mimic.

### Add copyright headers to two files

This one is trivial. In the previous build system (before we used autotools) two files (`main/mimic_voice_list.c` and `main/mimic_lang_list.c`) were autogenerated. When I switched to autotools I changed them so they were not autogenerated anymore (to make the build system simpler). They were lacking a copyright header and with my recent revision of copyright headers this was missing.

### Simplify tokenstream detection of emoji

While currently token to words routines do not handle emoji specifically (as far as I know), now emoji characters can be optionally placed into individual tokens with a single switch. While we don't have emotionally expressive speech synthesis if we ever work towards it we may benefit from this.
Basically we pass an extra 1 or 0 to the function that opens the tokenstream to determine if we want emoji characters in single tokens or not.

### CODE REVIEW

**Comment by forslund** [comment]
Looks good! Merging
